modules/user/user.controller.js

```diff
@@ -1,66 +1,109 @@
 const express = require("express");
 var router = express.Router(); // - /v1/user/
-
-const filePath = "./database/users.json";
 const jsonParser = express.json();
-const fs = require("fs");
-
-//получение всех пользователей
-router.get("/", function (req, res) {
-    fs.readFile(filePath, function (err, content) {
-        if (err) {
-            res.statusCode = 500;
-            res.end("Server error");
-            return;
-        }
+const { isEmpty, getUser } = require("../../utils.js");
+const userService = require("./user.service");
 
-        const users = JSON.parse(content);
+router.get("/", async function (req, res) {
+    try {
+        let users = await userService.readAllUsers();
```

Comment:
as I see, you do not change you variable user so use `const` instead of `let`

- [ ] replace all unchanged variables to `const`